packages/mobile/src/components/Chat/ChatDetail.tsx

```diff
@@ -299,9 +291,21 @@ export const ChatDetail = memo(
           presentationStyle: 'overFullScreen',
         });
         if (media.didCancel) return;
-        setAssets(media.assets ?? []);
+        const assets: ImageSource[] =
+          media.assets?.map((asset) => ({
```

Comment:
What exactly is happing there? The media.assets is already very close to `ImageSource[]` right? Perhaps better to add a wrapper around `launchCamera` that returns an ImageSource?

Reply:
It is close but still it throws a bunch of errors when mapping (height and width cannot be undefined) and then there's the type mapping we do to avoid breaking the app. Could create a helper method yess